frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -13,6 +13,7 @@ local T = require("ffi/util").template
 local ReaderDictionary = InputContainer:new{
     data_dir = nil,
     dict_window_list = {},
+    lookup_msg = _("Dictionary lookup of :\n“%1” ...")
```

Comment:
It's displayed as an InfoMessage while lookup is in progress, and discarded when done. I'm used to see "Looking up <text>...", but my "(Dictionnary|Wikipedia) lookup of <text>" sounds strange now.